mods/win32-tray-clock-experience.wh.cpp

```diff
@@ -0,0 +1,147 @@
+from Windows 10 and up. This *should* work on Windows 11 with the legacy
+taskbar, but I have not tested it.
+
+Based on the ExplorerPatcher implementation.
+
+**Preview**:
+
+![Preview](https://raw.githubusercontent.com/aubymori/images/main/win32-tray-clock-experience.png)
+*/
+// ==/WindhawkModReadme==
+
+#include <initguid.h>
+#include <windhawk_utils.h>
+#include <winerror.h>
+
+DEFINE_GUID(GUID_Win32Clock,
+    0x0A323554A,
+    0x0FE1, 0x4E49, 0xae, 0xe1,
+    0x67, 0x22, 0x46, 0x5d, 0x79, 0x9f
+);
+DEFINE_GUID(IID_Win32Clock,
+    0x7A5FCA8A,
+    0x76B1, 0x44C8, 0xa9, 0x7c,
+    0xe7, 0x17, 0x3c, 0xca, 0x5f, 0x4f
+);
+typedef interface Win32Clock Win32Clock;
+
+typedef struct Win32ClockVtbl
+{
+    BEGIN_INTERFACE
+
+        HRESULT(STDMETHODCALLTYPE* QueryInterface)(
+            Win32Clock* This,
+            /* [in] */ REFIID riid,
+            /* [annotation][iid_is][out] */
+            _COM_Outptr_  void** ppvObject);
+
+    ULONG(STDMETHODCALLTYPE* AddRef)(
+        Win32Clock* This);
+
+    ULONG(STDMETHODCALLTYPE* Release)(
+        Win32Clock* This);
+
+    HRESULT(STDMETHODCALLTYPE* ShowWin32Clock)(
+        Win32Clock* This,
+        /* [in] */ HWND hWnd,
+        /* [in] */ LPRECT lpRect);
+
+    END_INTERFACE
+} Win32ClockVtbl;
+
+interface Win32Clock
+{
+    CONST_VTBL struct Win32ClockVtbl* lpVtbl;
+};
+
+void (*ClockButton_StartTicking)(void *pThis);
+
+#define ClockButton_Window(pThis) *((HWND *)pThis + 1)
+
+HRESULT (*ClockButton_v_OnClick_orig)(void *, UINT);
+HRESULT ClockButton_v_OnClick_hook(
+    void *pThis,
+    UINT  uClickDevice
+)
+{
+    HRESULT hr = S_OK;
+    if (!FindWindowW(L"ClockFlyoutWindow", NULL))
+    {
+        Win32Clock *pClock = NULL;
+        hr = CoCreateInstance(
+            GUID_Win32Clock,
+            NULL,
+            CLSCTX_INPROC_SERVER | CLSCTX_INPROC_HANDLER,
+            IID_Win32Clock,
+            (void **)&pClock
+        );
+        if (SUCCEEDED(hr))
+        {
+            HWND hClock = ClockButton_Window(pThis);
+            RECT rc;
+            GetWindowRect(hClock, &rc);
+            hr = pClock->lpVtbl->ShowWin32Clock(
+                pClock,
+                hClock,
+                &rc
+            );
+            if (SUCCEEDED(hr))
+            {
+                ClockButton_StartTicking(pThis);
+            }
+            pClock->lpVtbl->Release(pClock);
+        }
+    }
+    return hr;
+}
+
+const WindhawkUtils::SYMBOL_HOOK hooks[] = {
+    {
+        {
+            L"public: void __cdecl ClockButton::StartTicking(void)"
+        },
+        &ClockButton_StartTicking,
+        nullptr,
+        false
+    },
+    {
+        {
+            L"protected: virtual long __cdecl ClockButton::v_OnClick(enum ClickDevice)"
+        },
+        &ClockButton_v_OnClick_orig,
+        ClockButton_v_OnClick_hook,
+        false
+    }
+};
+
+BOOL Wh_ModInit(void)
+{
+    if (!WindhawkUtils::HookSymbols(
+        GetModuleHandleW(NULL),
```

Comment:
For the old taskbar on Win11, you'll likely need to use `taskbar.dll`. Example:
https://github.com/ramensoftware/windhawk-mods/blob/f80c31b2a02a858d585c01b85fb3d5d5f1789555/mods/pinned-items-double-click.wh.cpp#L457-L470

Let me know if you want to add it or keep it as is, in which case it's probably better to update the description.